msteams-platform/m365-apps/specify-runtime-requirements.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Specify Microsoft 365 host runtime requirements in your app manifest
+description: Specify your app's runtime requirements in app manifest to provide a deliberate experience on supported Microsoft 365 hosts. 
+ms.date: 7/10/2024
+ms.author: mosdevdocs
+author: erikadoyle
+ms.topic: conceptual
+ms.localizationpriority: medium
+ms.subservice: m365apps
+---
+# Specify Microsoft 365 host runtime requirements in your app manifest
+
+> [!NOTE]
+>
+> The ability to specify Microsoft 365 host runtime requirements in app manifest (previously called Teams app manifest) is in [public developer preview](../resources/schema/manifest-schema-dev-preview.md).
+
+When you upgrade your Microsoft Teams personal tab or message extension app to use app manifest version 1.13 or later, it's available in other Microsoft 365 application hosts by default. However, if your app also includes components that aren't yet supported across other Microsoft 365 hosts, your app might only partially load, resulting in unplanned and unsupported user experiences.
+
+To ensure your app always provides customers with high quality experiences, you can specify your app's runtime requirements in app manifest to tailor its behavior in applicable Microsoft 365 hosts, or omit it from surfacing in contexts that you're not ready to support. Describing your app's runtime requirements in app manifest also helps ensure your production-ready app experience reaches wider audiences in additional Microsoft 365 host applications in the future.
```

Comment:
```suggestion
To ensure your app always provides customers with high-quality experiences, you can specify your app's runtime requirements in app manifest to tailor its behavior in applicable Microsoft 365 hosts, or omit it from surfacing in contexts that you're not ready to support. Describing your app's runtime requirements in app manifest also helps ensure your production-ready app experience reaches wider audiences in additional Microsoft 365 host applications in the future.
```

Reply:
Great suggestion. Small tweak:

To ensure high-quality app experiences while reaching the widest possible user audience, specify your app's runtime requirements in app manifest to tailor its behavior in applicable Microsoft 365 hosts, or omit it from surfacing in contexts you're not ready to support.

Reply:
That sounds even better, @erikadoyle. I've updated the suggestion.